version: "{{ docker_compose_file_version }}"
services:
  {{ plex_server_docker_name }}:
    depends_on:
      {{ arr_vpn_docker_name }}:
        condition: service_healthy
      {{ nginx_docker_name }}:
        condition: service_started
    environment:
        PGID: "{{ arr_gid }}"
        PUID: "{{ arr_uid }}"
        TZ: "US/Eastern"
        UMASK: "022"
        VERSION: latest
    healthcheck:
        interval: 20s
        retries: 3
        start_period: 60s
        test:
            - CMD
            - curl
            - --fail
            - --silent
            - --output
            - /dev/null
            - http://127.0.0.1:32400/identity
        timeout: 10s
    image: "{{ plex_server_docker_repo }}:{{ plex_server_docker_tag }}"
    logging: 
      driver: json-file
      options:
        max-size: "10m"
        max-file: "3"
    network_mode: host
    restart: always
    volumes:
        - "{{ plex_server_base_dir }}:/config/"
{% for item in plex_server_data_dirs %}
        - "{{ item }}:{{ item }}:ro"
{% endfor %}